<launch>

  <param name = "use_sim_time" value = "true" />
  <arg name="bagfile_path"  value="$(env ROS_DATASET_PATH)/temp/vicon_2015-06-28-17-13-56.bag" />

  <!--
  <arg name="bagfile_path"  value="$(env ROS_DATASET_PATH)/statefarm_dataset/mrsl03.bag" />

  -->
  <node name="rosbag"
        pkg="rosbag"
        type="play"
        output="screen" 
        args = "--loop --rate=1.0 --start=0 --duration=999 --clock $(arg bagfile_path)">
  </node>

  <node name="odom_visualization"
        pkg ="odom_visualization"
        type="odom_visualization_node"
        output="screen" >

    <remap from="~odom" to="/vicon/InsKHex_odom"/>
    <rosparam file="$(find odom_visualization)/config/config01.yaml" />

  </node>

  <node	pkg="rviz"
		name="rviz"
		type="rviz"
		output="screen"
    args  ="-d $(find odom_visualization)/rviz/odom_visualization01.rviz">
		<param name="fixed-frame"	 value="world"/>
</node>

</launch>
